complex_project/my_complex.h

```diff
@@ -1,85 +1,78 @@
-//
-// Created by Sandi on 10/09/2020.
-//
 
-#ifndef TEST1_MY_COMPLEX_H
-#define TEST1_MY_COMPLEX_H
+#pragma once
 
 #include <iostream>
+#include <cmath>
 
-class MyComplex
-{
-private:
-    double real;
-    double imag;
-public:
-    MyComplex(): real(0.0), imag(0.0) {}
-    MyComplex(double real): real(real), imag(0.0) {}
-    MyComplex(double real, double imag): real(real), imag(imag) {}
-
-    // Copy constructor
-    MyComplex(const MyComplex&C)
-    {
-        real = C.real;
-        imag = C.imag;
-    }
+struct MyComplex {
+    /** This is my complex object, equality was not defined in the exercise.
+     *  this is my interpretation of equality.
+     */
```

Comment:
Formatting:
```suggestion
    /**
     * This is my complex object, equality was not defined in the exercise.
     *  this is my interpretation of equality.
     */
```

But actually I think it makes sense to define two complex numbers as equal iff the real and imaginary part are equal in the usual `==` sense, with no separate threshold checking.